index.html

```diff
@@ -859,5 +859,94 @@ <h2>Internationalization Considerations</h2>
+      </pre>
+      <pre class="example nohighlight" title="JSON Web Token Status List (Verified)">
+{
+  "protectedHeader": {
+    "alg": "ES256",
+    "iss": "did:example:123",
+    "kid": "#0",
+    "typ": "vc+ld+jwt",
+    "cty": "vc+ld+json"
+  },
+  "payload": {
+    "@context": [
+      "https://www.w3.org/ns/credentials/v2",
+      "https://w3id.org/vc/status-list/2021/v1"
+    ],
+    "id": "https://vendor.example/credentials/status/3",
+    "type": [
+      "VerifiableCredential",
+      "StatusList2021Credential"
+    ],
+    "issuer": "did:example:123",
+    "validFrom": "2021-04-05T14:27:40.000Z",
+    "credentialSubject": {
+      "id": "https://vendor.example/credentials/status/3#list",
+      "type": "StatusList2021",
+      "statusPurpose": "suspension",
+      "encodedList": "H4sIAAAAAAAAA2MAAI3vAtIBAAAA"
+    }
+  }
+}
+      </pre>
+      <pre class="example nohighlight" title="JSON Web Token Verifiable Credential With Status">
+eyJhbGciOiJFUzI1NiIsImlzcyI6ImRpZDpleGFtcGxlOjEyMyIsImtpZCI6IiMwIiwidHlwIjoidmMrbGQrand0IiwiY3R5IjoidmMrbGQranNvbiJ9.eyJAY29udGV4dCI6WyJodHRwczovL3d3dy53My5vcmcvbnMvY3JlZGVudGlhbHMvdjIiLCJodHRwczovL3czaWQub3JnL3ZjL3N0YXR1cy1saXN0LzIwMjEvdjEiXSwiaWQiOiJodHRwOi8vZXhhbXBsZS5jb20vY3JlZGVudGlhbHMvMTg3MiIsInR5cGUiOlsiVmVyaWZpYWJsZUNyZWRlbnRpYWwiLCJOZXdDcmVkZW50aWFsVHlwZSJdLCJpc3N1ZXIiOnsiaWQiOiJkaWQ6ZXhhbXBsZToxMjMiLCJ0eXBlIjpbIk9yZ2FuaXphdGlvbiIsIk9yZ2FuaXphdGlvblR5cGUiXX0sInZhbGlkRnJvbSI6IjIwMTAtMDEtMDFUMTk6MjM6MjRaIiwiY3JlZGVudGlhbFN0YXR1cyI6eyJpZCI6Imh0dHBzOi8vZXhhbXBsZS5jb20vY3JlZGVudGlhbHMvc3RhdHVzLzMjNCIsInR5cGUiOiJTdGF0dXNMaXN0MjAyMUVudHJ5Iiwic3RhdHVzUHVycG9zZSI6InN1c3BlbnNpb24iLCJzdGF0dXNMaXN0SW5kZXgiOiI0Iiwic3RhdHVzTGlzdENyZWRlbnRpYWwiOiJodHRwczovL2V4YW1wbGUuY29tL2NyZWRlbnRpYWxzL3N0YXR1cy8zIn0sImNyZWRlbnRpYWxTdWJqZWN0Ijp7ImlkIjoiZGlkOmV4YW1wbGU6NDU2IiwidHlwZSI6WyJQZXJzb24iLCJKb2JUeXBlIl0sImNsYWltTmFtZSI6ImNsYWltVmFsdWUifX0.krP-9305AZEUu686iw_DfakBihBYPbyXDqQ51n2eYLJolg04k0RFS7g-JCKbpRkUclqQKrN3OBs2SQdcMgGpWg
+      </pre>
+      <pre class="example nohighlight" title="JSON Web Token Verifiable Credential With Status (Verified)">
+{
+  "suspension": false,
+  "protectedHeader": {
+    "alg": "ES256",
+    "iss": "did:example:123",
+    "kid": "#0",
+    "typ": "vc+ld+jwt",
+    "cty": "vc+ld+json"
+  },
+  "payload": {
+    "@context": [
+      "https://www.w3.org/ns/credentials/v2",
+      "https://w3id.org/vc/status-list/2021/v1"
+    ],
+    "id": "http://example.com/credentials/1872",
+    "type": [
+      "VerifiableCredential",
+      "NewCredentialType"
+    ],
+    "issuer": {
+      "id": "did:example:123",
+      "type": [
+        "Organization",
+        "OrganizationType"
```

Comment:
These are also undefined, right?

Reply:
They are not "undefined", they are "vocab defined".